master/buildbot/reporters/github.py

```diff
@@ -126,50 +126,53 @@ def send(self, build):
         context = yield props.render(self.context)
 
         sourcestamps = build['buildset'].get('sourcestamps')
-
-        if not sourcestamps or not sourcestamps[0]:
+        if not sourcestamps:
             return
 
-        project = sourcestamps[0]['project']
+        for sourcestamp in sourcestamps:
+            project = sourcestamp['project']
 
-        branch = props['branch']
-        m = re.search(r"refs/pull/([0-9]*)/merge", branch)
-        if m:
-            issue = m.group(1)
-        else:
+            # If branch is None, so is issue
             issue = None
+            if 'branch' in props:
+                m = re.search(r"refs/pull/([0-9]*)/merge", props['branch'])
+                if m:
+                    issue = m.group(1)
+
+            repoOwner = None
```

Comment:
i understand we follow Twisted naming scheme in a lot of places, there's no need to do it here: `repo_owner` would be just fine

Reply:
Fixed now.